test/verify/run-tests

```diff
@@ -83,10 +83,16 @@ def finish_test(opts, test):
             print_test(test, print_tap=False)
             return None, 0
 
-    if test.retries < 2 and b"# RETRY" in test.output:
-        retry_reason = re.search(b"# RETRY (.*)$", test.output, re.MULTILINE).group(1)
+    if test.retries < 2:
+        # do we get a specific retry reason from tests-policy?
+        m = re.search(b"# RETRY (.*)$", test.output, re.MULTILINE)
+        if m:
+            retry_reason = m.group(1)
+        else:
+            # HACK: many tests are unstable, always retry them 3 times
+            retry_reason = b"be robust against unstable tests"
         test.retries += 1
-        test.output += " # RETRY {0}\n".format(test.retries).encode()
+        test.output += b" # RETRY %i (%s)\n" % (test.retries, retry_reason)
```

Comment:
FTR: `bytes` does not have a `.format()` option, but `%s` works with bytes as well.